packages/voting/graphql/resolvers/_queries/votings.js

```diff
@@ -0,0 +1,18 @@
+module.exports = async (_, args, context) => {
+  return [
+    {
+      id: '47912a28-b6a2-11e8-b23e-784f4383ef45',
+      slug: 'gen18-budget',
+      beginDate: new Date('2018-10-17T00:00:00.000Z'),
+      endDate: new Date('2018-10-31T00:00:00.000Z'),
+      options: []
+    },
+    {
+      id: 'ca4220e4-b6a2-11e8-8ba4-784f4383ef45',
+      slug: 'gen18-audit',
+      beginDate: new Date('2018-10-17T00:00:00.000Z'),
+      endDate: new Date('2018-10-31T00:00:00.000Z'),
+      options: []
+    }
+  ]
+}
```

Comment:
please remove mockup data before deploying to production. Either provide a `create` mutation or a insert script